src/components/Questions/styles.scss

```diff
@@ -0,0 +1,3 @@
+.align-center {
```

Comment:
CSS classes should describe what some component is: a class could be e.g. title, title icon, description, box-header